src/peerconnection.cpp

```diff
@@ -544,46 +544,47 @@ void PeerConnection::forwardMedia(message_ptr message) {
 	if (!message)
 		return;
 
-	if (message->type == Message::Type::Control) {
-		std::shared_lock lock(mTracksMutex); // read-only
-		for (auto it = mTracks.begin(); it != mTracks.end(); ++it)
-			if (auto track = it->second.lock())
-				return track->incoming(message);
-
-		PLOG_WARNING << "No track available to receive control, dropping";
-		return;
-	}
-
-	unsigned int payloadType = message->stream;
+//	if (message->type == Message::Type::Control) {
+//		std::shared_lock lock(mTracksMutex); // read-only
+//		for (auto it = mTracks.begin(); it != mTracks.end(); ++it)
+//			if (auto track = it->second.lock())
+//				return track->incoming(message);
+//
+//		PLOG_WARNING << "No track available to receive control, dropping";
+//		return;
+//	}
```

Comment:
You may clean up this block, it's now useless thanks to your modification to use SSRC.